app/views/thredded/messageboards/_messageboard.html.erb

```diff
@@ -19,6 +19,8 @@
                 user: messageboard.last_user.thredded_display_name %>
         </p>
       <% end %>
+      <br>
+      <%= inline_svg('thredded/lock.svg', class: 'thredded--messageboard--icon') if messageboard.locked %>
```

Comment:
How about `thredded.messageboard.locked_notice` key for the tooltip?

Reply:
Cool, just left to decide for the placement and the translations. Should I push with the latest screenshot that I posted?